### DISCUSSION TITLE
Feature request: Allow shutter position in 0.1% steps.

### DISCUSSION THREAD

**Discussion by s0170071** [arendst/Tasmota#14108] [Ideas]
Feature request: Allow shutter position in 0.1% steps. 
Why? 
1. My Homematic Rollershutter devices do it already. I use it to move venetian blinds to 99.5% when I open the window, which tilts 45°
2. I want that to be a one-time tilt action. The built-in tilt option is designed to keep the tilt and it kicks in whenever I move the blind.

Alternatively: option to disable the auto-tilt after moving the blinds.

**Comment by stefanbode** [comment]
changing is to 0.1% will not do any benefit. The problem is internally, that the minimum timeslice is 0.05 seconds. Nothing can be faster. This does mean if you want to start and stop something this will at least 0.2 sec. And believe me your motors do not like it. In theory this will all work but not in the wild life. The restore of the tilt after moving is a feature because you change the positon and not the tilt. You can add a rule that changes also the tilt. Then it will take the tilt you set in your rules.

**Comment by s0170071** [comment]
My blinds take 120s to move from top to bottom. 1% means 1.2s, 0
1% are 120ms. Sounds reasonable to me



Am 5. Februar 2022 12:49:55 schrieb stefanbode ***@***.***>:
